boot/assets.go

```diff
@@ -257,11 +257,11 @@ type TrustedAssetsInstallObserver struct {
 
 	blName        string
 	managedAssets []string
-	trustedAssets []string
+	trustedAssets map[string]string
```

Comment:
probably a good idea to add a quick doc comment about what is the key and value of this